### DISCUSSION TITLE
FEATURE: 5.0版本开发计划咨询

### DISCUSSION THREAD

**Discussion by openwj** [vbenjs/vue-vben-admin#4023] [General]
### Version

Vben Admin Pro

### Description

5.0版本变化好大呀，感觉非常 nice!。请问后续支持 像 2.0  + 里面的组件封装吗，这个具体啥时候上线。目前官方演示，没有 放出 基础 的例如常用 的 Form,Table 等等
![截图20240802091747](https://github.com/user-attachments/assets/c2fbfd75-e0fc-4d9b-abf4-f17209be06cb)

### Proposed Solution

增加功能性组件

### Validations

- [X] Read the [docs](https://anncwb.github.io/vue-vben-admin-doc/)
- [X] Ensure the code is up to date. (Some issues have been fixed in the latest version)
- [X] I have searched the [existing issues](https://github.com/vbenjs/vue-vben-admin/issues) and checked that my issue does not duplicate any existing issues.

**Comment by likui628** [comment]
[Roadmap
](https://github.com/orgs/vbenjs/projects/5/views/1)
